### ISSUE TITLE
A11y - no Quick Fix avaiability hint

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#138]
## Problem

Quick Fix availability should be hinted by screen readers while going through issue list.

Right now the only way to know this is to tab out from navigation region, and see what are the next focusable items. This is not a good design.
## Proposal

I belive it should be placed between issue title and issue description.

Currently the speak order is following:
1. Navigation info (Issue x of x)
2. Issue title
3. Issue descr

So it would make sense to put a text "Quick Fix available" if there is QF available, or skip extra text if there is no QF.

It is important that the isse title will be spoken before QF as it is more important information:
- User should know issue type ASAP.
- User familiar with AC will know if QF is available knowing the issue type.
